### PR TITLE
Adds the Japanese and Latin Languages

### PR DESCRIPTION
**Pull request by Honestly101**
## About The Pull Request

This PR adds two new languages Neohongo (japanese) and Ancient Latin.

## Why It's Good For The Game

Eris has very few languages and i thought it would be nice for NT to have Ancient Latin as a secondary language and Moebius get Japanese since they are a bunch of nerds
## Changelog
:cl:
add: Added two new languages
/:cl:

### CODE REVIEW

File: code/modules/mob/language/ancient_latin.dm [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/datum/language/ancientlatin
+	name = LANGUAGE_LATIN
+	desc = "An ancient language once used by most of europe but now only followers of Neotheology speak and understand"
+	colour = "latin"
+	key = "l"
+	space_chance = 80
+	shorthand = "LA"
+	syllables = list("a preiori", "curriclum vitae", "ex nihilo", "in toto", "ipso facto", "non sequiter", \
```
Comment (infraredbaron) [inline code/modules/mob/language/ancient_latin.dm:8]: You missed perfect opportunity to use Lorem Ipsum

**Review by infraredbaron — APPROVED** [review]
I don't see any mistakes, so it should be good.

**Comment by Honestly101** [comment]
yeah dont worry, i compiled it and play tested and all works as intended

**Comment by Alanii** [comment]
Why?

**Comment by Alanii** [comment]
Instead of editing comment i make new.

If you are going to make japanese language do not use whole words. Use syllables and the like.

Using whole words is just asking for a 2chan post where a bunch of NEET talk shit about crappy language and how awful and lazy western people are when comes to languages.

This also look like joke PR with japanese. It hurts my head when I put these together and you read them.

**Comment by Honestly101** [comment]
Unless you didn't know the other languages in Eris use whole words, so you might aswell complain about those ones aswell. Just for you to know i intended for Japanese to be cursed as hell when used in eris, you ain't meant to understand it in the first place anyway, so just live with it.

**Comment by Alanii** [comment]
> Unless you didn't know the other languages in Eris use whole words, so you might aswell complain about those ones aswell. I
> intended for Japanese to be cursed as hell when used in eris, you ain't meant to understand it in the first place anyway, so just live with it.

You have literal words such as cute, pervert, senior, "you are already dead" phrase, you have honorifics, if you want to make a meme language I do not think Eris is the place for it.

The russian language isnt as memed as hard as this one.

**Comment by Honestly101** [comment]
Well if you dont like it being a meme language by all means change it yourself, no one is stopping you.

**Review by Firefox13 — CHANGES_REQUESTED** [review]
Alanii has a point - while other(particularily techno-russian) languages have less-serious words, you're overdoing the japanese one, especially because it'll be memed much more due to weebs. Lower the amount of meme words; consult with Alanii since they can actually talk japanese.

File: code/datums/setup_option/backgrounds/origin.dm [hunk 2]
```diff
@@ -94,6 +101,10 @@
 		STAT_COG = 5
 	)
 
+/datum/category_item/setup_option/background/origin/oberth/apply(mob/living/carbon/human/character)
```
Comment (LethalGhost) [inline code/datums/setup_option/backgrounds/origin.dm:104]:
```suggestion
/datum/category_item/setup_option/background/origin/shimatengoku/apply(mob/living/carbon/human/character)
```

**Review by LethalGhost — CHANGES_REQUESTED** [review]

File: code/datums/setup_option/backgrounds/origin.dm (hunk 2 above)
Comment (LethalGhost) [reply code/datums/setup_option/backgrounds/origin.dm:104]: Also. Why all new languages have 100% chance to be added to character?

**Comment by Honestly101** [comment]
OK, imma take down this PR and when everything is trully i will make another PR.

**Pull request closed without merging**